### PR TITLE
877 - resize register list height

### LINKED ISSUE #877: Quest : Pagination for search results not loading all results
**Describe the bug**
All search results that are above the pagination limit should be displayed on separate pages in the app. Not all results are currently displayed despite viewing the correct search count. See screen shot  

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Quest app 
2. Search for Mussa (Currently has 9 results)
3. Scroll down to on search results page 
4. Only 6 patient can be viewed on the search result page 

**Expected behaviour**
View the correct number of results and if more than the pagination limit vie results on different result pages
 
**Screenshots**

**Smartphone (please complete the following information):**
 - Physical device or an emulator 

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by AbdulWahabMemon**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #877

**Type**
Choose one: Bug fix

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/ComposeRegisterFragment.kt [hunk 1]
```diff
@@ -48,21 +48,23 @@ abstract class ComposeRegisterFragment<I : Any, O : Any> : BaseRegisterFragment<
           val pagingItems = registerData.value.collectAsLazyPagingItems()
           val showResultsCount by registerDataViewModel.showResultsCount.observeAsState(false)
           val showLoader by registerDataViewModel.showLoader.observeAsState(false)
-
-          if (showLoader) LoaderDialog(modifier = Modifier)
+          val _modifier = Modifier
```
Comment (ellykits) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/ComposeRegisterFragment.kt:51]: Rename this variable to **modifier** to follow kotlin conventions

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PaginatedRegisterViews.kt [hunk 2]
```diff
@@ -178,32 +181,64 @@ fun PaginatedRegister(
...
-      modifier = modifier.weight(1f).padding(4.dp).fillMaxSize()
-    ) {
-      if (loadState == LoadState.Loading) {
-        CircularProgressBar()
-      } else {
-        if (resultCount == 0 && showResultsCount) {
-          NoResults(modifier = modifier)
+  Column(modifier = modifier.fillMaxWidth().height(200.dp)) {
+    ConstraintLayout(modifier = Modifier.fillMaxSize()) {
+      val (_top, _body, _bottom, _b2) = createRefs()
```
Comment (ellykits) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PaginatedRegisterViews.kt:186]: Remove underscore in destructed data class properties names unless those are the actual names. Rename **_b2** appropriately. Underscores are conventionally used in field properties to denote them as private. Sometimes they are used as placeholders for unused properties.

**Review by ellykits — CHANGES_REQUESTED** [review]
Register list in ANC application seems to be broken (not loading any data).

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/ComposeRegisterFragment.kt (hunk 1 above)
Comment (AbdulWahabMemon) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/ComposeRegisterFragment.kt:51]: modifier is required as its passing to internal components also i.e body() here, I have renamed _modifier -> modifier

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PaginatedRegisterViews.kt (hunk 2 above)
Comment (AbdulWahabMemon) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PaginatedRegisterViews.kt]: updated

**Review by ellykits — APPROVED** [review]

**Pull request merged**